src/Metadata/AmazonMetadataBuilder.php

```diff
@@ -116,6 +117,18 @@ protected function getDefaultMetadata()
      */
     protected function getContentType($filename)
     {
+        // NEXT_MAJOR: Remove this BC hack
+        if (class_exists(MimeUtil::class)) {
```

Comment:
We can use a symfony component here, one it got released:

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Mime/MimeTypes.php